app/helpers.php

```diff
@@ -184,5 +184,5 @@ function is_local_environment()
 
 function currentUserHasAuth()
 {
-    return Auth::check() || Auth::guard('vatsim-sso')->check();
+    return Auth::check() && Auth::guard('vatsim-sso')->check() && Auth::guard('web')->check();
```

Comment:
`Auth::check()` should default  to using the default guard, which is `web` in our case

Also, users won't be authenticated in multiple guards, so the `||` operator is required. I.e we want to show the logout button to people who have been authed through CERT as well as those who have also passed secondary authentication.